dicom4ortho/model.py

```diff
@@ -250,6 +250,21 @@ def copy_mwl_tags(self, dicom_mwl=None):
         if 'AdmittingDiagnosesCodeSequence' in self.dicom_mwl:
             self._ds.AdmittingDiagnosesCodeSequence = self.dicom_mwl.AdmittingDiagnosesCodeSequence
 
+        # RequestingPhysician, ReferringPhysicianName, OperatorsName, ScheduledPerformingPhysicianName
+        if 'RequestingPhysician' in self.dicom_mwl:
+            self._ds.RequestingPhysician = self.dicom_mwl.RequestingPhysician
+
+        if 'ReferringPhysicianName' in self.dicom_mwl:
+            self._ds.ReferringPhysicianName = self.dicom_mwl.ReferringPhysicianName
+
+        if 'OperatorsName' in self.dicom_mwl:
+            self._ds.OperatorsName = self.dicom_mwl.OperatorsName
+
+        if 'ScheduledProcedureStepSequence' in self.dicom_mwl and len(self.dicom_mwl.ScheduledProcedureStepSequence) > 0:
+            scheduledPerformingPhysicianName = self.dicom_mwl.ScheduledProcedureStepSequence[0].ScheduledPerformingPhysicianName
+            for step in self._ds.RequestAttributesSequence:
```

Comment:
Document the reason to add an attribute in the RequestedAttributeSequence here, instead of in `_set_request_attributes()`  written purposely to do this task.

Why did you call them "step"? Took me a while to figure out this is probably just a poor name choice. You are looping through a sequence of request attributes. So calling this `request_attribute` or just `attribute`, would make the life of the code reader easier.